pkg/monitor/cluster/clusterwideproxystatus.go

```diff
@@ -0,0 +1,238 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"net/url"
+	"strconv"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/network/armnetwork/v2"
+	"github.com/Azure/go-autorest/autorest/azure"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	apisubnet "github.com/Azure/ARO-RP/pkg/api/util/subnet"
+	arov1alpha1 "github.com/Azure/ARO-RP/pkg/operator/apis/aro.openshift.io/v1alpha1"
+)
+
+const (
+	cwp                  = "clusterWideProxy.status"
+	cwpErrorMessage      = "NoProxy entries are incorrect"
+	cluster              = "cluster"
+	mandatory_no_proxies = "localhost,127.0.0.1,.svc,.cluster.local,168.63.129.16"
+	AzureDNS             = "169.254.169.254"
```

Comment:
To make my suggestion about these IPs clear:

```suggestion
	mandatory_no_proxies = "localhost,127.0.0.1,.svc,.cluster.local,169.254.169.254"
	AzureDNS             = "168.63.129.16"
```

Reply:
updated